src/components/CvViewer/CvViewer.stub.ts

```diff
@@ -10,35 +10,41 @@ export const cvDataMock: ResumeViewerType = {
     address: 'Brest, Belarus',
```

Comment:
I suggest getting rid of some specific stub data, especially when it is related to some sensitive data, we need to put there more common data

Reply:
Yeap, makes sense